.github/CODEOWNERS

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2022 Apple Inc. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1.  Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2.  Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS BE LIABLE FOR ANY
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+# ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# ================================================================================
+#
+#     Contributors do not "own" WebKit components. This file is used to
+#     automatically add reviewers to pull requests.
+#
+# ================================================================================
+
+/.github @JonWBedard
+/metadata @JonWBedard
+
+# ================================================================================
+/Tools @JonWBedard
+
+/Tools/CISupport @aj062 @JonWBedard @ryanhaddad
```

Comment:
We can't do that for every combination of people. The team here is more generically "WebKit Operations", but not everyone on that team should be CCed as a reviewer for buildbot changes. In other directories (Like `/Tools`), it might be appropriate to CC the whole team, but something like `/Tools/WebKitTestRunner` probably also needs product engineers.

I think in cases where we don't have a clear team, or the entire team isn't appropriate to CC, we should use individuals rather than have a proliferation of fragmented teams.